cmd/monaco/download/download_command.go

```diff
@@ -48,7 +48,7 @@ To download entities, use download entities`,
 
 	GetDownloadConfigsCommand(fs, command, downloadCmd)
 
-	if utilEnv.FeatureFlagEnabled("MONACO_FEAT_ENTITIES") {
+	if featureflags.FeatureFlagEnabled("MONACO_FEAT_ENTITIES") {
```

Comment:
nitpick: just reading this, this could probably be called as 
```suggestion
	if featureflag.Enabled("MONACO_FEAT_ENTITIES") {
```
and tell me everything I need to know with a simpler package and function name

Reply:
i also thought about exactly naming it like that first. :D 
my gut feeling tells me that `featureFlag.IsFeatureEnabled()` is the most common.